spec/yaml/swiftnav/sbp/tracking.yaml

```diff
@@ -50,7 +50,7 @@ definitions:
     short_desc: Satellite tracking channel states
     desc: |
       The tracking message returns a variable-length array of tracking
-      channel states. It reports status and snr power measurements for
+      channel states. It reports status and C/N0 measurements for
```

Comment:
We haven't use `C/N0` in the documentation - have used instead `Carrier-to-Noise density`.